app/config-setup.py

```diff
@@ -0,0 +1,28 @@
+import json
+import requests
+
+open('/home/pi/config.json', 'w').write('{}')
```

Comment:
In general, it's a good idea never to have stuff happen in a module, but outside functions.

This is because if someone wanted to `import config-setup`, surprising things might happen without them having done anything other than the import.

In this specific case, of course, it doesn't matter much because it's unlikely anyone would ever import this script.

Still, it's a good idea to get into good habits.

A good pattern would be to refactor this script so that the `get_box_id()` method just *returns* the id; call that function from `main()` and store the result in a variabe like `box_id` (i.e. `box_id = get_box_id()`); then pass that to a `set_box_id(box_id)` function which opens the config file and writes to it.  This function could also check if a file already exists, and says "are you sure" if it does, perhaps?